courses/free-courses-id.md

```diff
@@ -219,7 +219,8 @@
 * [ReactJS Untuk Pemula](https://www.petanikode.com/reactjs-untuk-pemula/) - Petani Kode
```

Comment:
Using Prawito Hudoro's Youtube playlist where video `frwlr7uh0io` belongs to.
```suggestion
* [ReactJS Untuk Pemula](https://www.petanikode.com/reactjs-untuk-pemula/) - Petani Kode
* [ReactJS-Firebase Tutorial](https://www.youtube.com/playlist?list=PLU4DS8KR-LJ2CnIvj7tI0zoijDSgR1m9j) - Prawito Hudoro
```

Reply:
Solved at https://github.com/EbookFoundation/free-programming-books/pull/6313/commits/5e861a4e6d350e39626173986741e614947ce56f